services/security/identity/identity.go

```diff
@@ -58,7 +58,7 @@ func getWssdIdentity(id *security.Identity) (*wssdcloudsecurity.Identity, error)
 		return nil, errors.Wrapf(errors.InvalidInput, "Identity name is missing")
 	}
 
-	if !(filepath.IsAbs(*id.LoginFilePath)) {
+	if id.LoginFilePath != nil && !(filepath.IsAbs(*id.LoginFilePath)) {
```

Comment:
I believe this can be null since if we don't `enableTokenAutoRotate` then we don't need the `LoginFilePath` to be populated. We have a check at the `moc-cli` level[1]. 

I'm not sure whether we need to have another check at the moc-sdk level. 
[1]https://github.com/microsoft/moccli/blob/master/cmd/mocctl/security/identity/create/create.go#:~:text=if%20flags.AutoRotate%20!%3D%20DefaultAutoRotate

Reply:
@raghavendra-nataraj is this an optional param?

Reply:
It is optional as auto rotate can happen with loginpath. This is what we do for cloudoperator today